src/main/java/org/junit/rules/ExternalResource.java

```diff
@@ -77,8 +77,10 @@ protected void before() throws Throwable {
 
     /**
      * Override to tear down your specific external resource.
+     *
+     * @throws Exception if teardown fails
      */
-    protected void after() {
+    protected void after() throws Exception {
```

Comment:
I don't think that closing a resource usually involves doing reflection or anything that throws Throwable, but anyway yeah, it's harmless, so I'll do.